### PR TITLE
some changes

### PR DESCRIPTION
**Pull request by leo-ship-it**

### CODE REVIEW

File: water_neuron/src/main.rs [hunk 1]
```diff
@@ -364,7 +365,7 @@ fn http_request(req: HttpRequest) -> HttpResponse {
                 )?;
                 w.encode_gauge(
                     "stakers",
-                    s.principal_to_deposit.keys().len() as f64,
+                    s.account_to_deposits.keys().len() as f64,
```
Comment (uk4z) [inline water_neuron/src/main.rs:368]: account_to_deposit without s

File: water_neuron/src/state.rs [hunk 2]
```diff
@@ -183,8 +183,8 @@ pub struct State {
     pub transfer_executed: BTreeMap<TransferId, ExecutedTransfer>,
 
     // Maps for tracking purposes.
-    pub principal_to_deposit: BTreeMap<Principal, Vec<TransferId>>,
-    pub principal_to_withdrawal: BTreeMap<Principal, Vec<WithdrawalId>>,
+    pub account_to_deposits: BTreeMap<Account, Vec<TransferId>>,
```
Comment (uk4z) [inline water_neuron/src/state.rs:186]: without s

**Review by uk4z — COMMENTED** [review]
lgtm

File: water_neuron/src/dashboard.rs [hunk 3]
```diff
@@ -192,12 +192,37 @@ fn construct_metadata_table() -> String {
                         <td>{}</td>
                     </tr>
                     <tr>
-                        <th>6-month nICP Neuron</th>
-                        <td>{} Fetched {} ICP - Tracked {} ICP</td>
+                        <th rowspan=\"4\">6-month nICP Neuron</th>
```
Comment (EnzoPlayer0ne) [inline water_neuron/src/dashboard.rs:195]: month -> months

File: water_neuron/src/dashboard.rs [hunk 4]
```diff
@@ -192,12 +192,37 @@ fn construct_metadata_table() -> String {
...
+                    <tr>
+                        <td>Tracked {} ICP</td>
+                    </tr>
+                    <tr>
+                        <td>
+                            <a target=\"_blank\" href=\"https://dashboard.internetcomputer.org/account/11db16e8da65784bc03feafc67fde5317bbbe59852bd6d9af8f8753e8c8e3279\">Maturity Account</a>
+                        </td>
+                    </tr>
+                    <tr>
+                        <th rowspan=\"3\">8-year SNS Neuron</th>
```
Comment (EnzoPlayer0ne) [inline water_neuron/src/dashboard.rs:210]: Pareil non?

**Review by EnzoPlayer0ne — APPROVED** [review]

File: water_neuron/src/dashboard.rs (hunk 3 above)
Comment (leo-ship-it) [reply water_neuron/src/dashboard.rs:195]: https://dictionary.cambridge.org/example/english/six-month-term

File: water_neuron/src/dashboard.rs (hunk 4 above)
Comment (leo-ship-it) [reply water_neuron/src/dashboard.rs:210]: eight years / eight-year

**Review by uk4z — APPROVED** [review]

**Pull request merged**